Modelica/Magnetic/FluxTubes/Material/SoftMagnetic/CobaltIron/Vacoflux50.mo

```diff
@@ -1,12 +1,14 @@
 within Modelica.Magnetic.FluxTubes.Material.SoftMagnetic.CobaltIron;
 record Vacoflux50 "Vacoflux 50 (50% CoFe)"
   extends FluxTubes.Material.SoftMagnetic.BaseData(
+    label="Vacoflux 50",
```

Comment:
```suggestion
record Vacoflux50 "VACOFLUX 50 (50% CoFe)"
  extends FluxTubes.Material.SoftMagnetic.BaseData(
    label="VACOFLUX 50",
```

See https://vacuumschmelze.com/Company/Quality-and-Safety/Information-for-Manufactured-Articles/VACOFLUX